### PR TITLE
Update recommended models for 2 tasks

### PR DESCRIPTION
**Pull request by NielsRogge**
Some minor updates for the recommended models of the "image segmentation" and "object detection" tasks:

* "facebook/detr-resnet-50-panoptic" is pretty outdated and also not that good
* "microsoft/beit-base-patch16-224-pt22k-ft22k" was linked in the object detection page, but it's an image classification model

cc @merveenoyan

### CODE REVIEW

**Review by Vaibhavs10 — COMMENTED** [review]
JFYI - @merveenoyan is updating these in this PR: https://github.com/huggingface/huggingface.js/pull/876

**Comment by merveenoyan** [comment]
My PR is merged already, @NielsRogge can you lint?

File: packages/tasks/src/tasks/object-detection/data.ts [hunk 1]
```diff
@@ -43,15 +43,16 @@ const taskData: TaskDataCustom = {
...
 		{
-			description: "Strong object detection model trained on ImageNet-21k dataset.",
-			id: "microsoft/beit-base-patch16-224-pt22k-ft22k",
+			description: "YOLOv10 is great model for real-time and accurate object detection.",
+			id: "jameslahm/yolov10x",
 		},
 		{
-			description: "Fast and accurate object detection model trained on COCO dataset.",
+			description:
+				"RT-DETR is a fast and accurate object detection model pre-trained on the COCO and Object365 datasets.",
```
Comment (merveenoyan) [inline packages/tasks/src/tasks/object-detection/data.ts:55]:
```suggestion
				"Fast and accurate object detection model trained on COCO and Object365 datasets.",
```

File: packages/tasks/src/tasks/object-detection/data.ts [hunk 2]
```diff
@@ -43,15 +43,16 @@ const taskData: TaskDataCustom = {
...
 	models: [
 		{
-			description: "Solid object detection model trained on the benchmark dataset COCO 2017.",
+			description: "Solid object detection model pre-trained on the COCO 2017 dataset.",
 			id: "facebook/detr-resnet-50",
 		},
 		{
-			description: "Strong object detection model trained on ImageNet-21k dataset.",
-			id: "microsoft/beit-base-patch16-224-pt22k-ft22k",
+			description: "YOLOv10 is great model for real-time and accurate object detection.",
```
Comment (merveenoyan) [inline packages/tasks/src/tasks/object-detection/data.ts:50]:
```suggestion
			description: "Real-time and accurate object detection model.",
```

File: packages/tasks/src/tasks/image-segmentation/data.ts [hunk 3]
```diff
@@ -44,21 +44,22 @@ const taskData: TaskDataCustom = {
...
-		{
-			description: "Semantic segmentation model trained on ADE20k benchmark dataset with 512x512 resolution.",
-			id: "nvidia/segformer-b0-finetuned-ade-512-512",
-		},
 		{
 			description: "A multipurpose image segmentation model for high resolution images.",
 			id: "ZhengPeng7/BiRefNet",
 		},
+		{
+			description: "Semantic segmentation model fine-tuned on ADE20k at 512x512 resolution.",
```
Comment (merveenoyan) [inline packages/tasks/src/tasks/image-segmentation/data.ts:60]:
```suggestion
			description: "Semantic segmentation model trained on ADE20k dataset.",
```

File: packages/tasks/src/tasks/image-segmentation/data.ts [hunk 4]
```diff
@@ -44,21 +44,22 @@ const taskData: TaskDataCustom = {
 	models: [
 		{
 			// TO DO: write description
-			description: "Solid panoptic segmentation model trained on the COCO 2017 benchmark dataset.",
-			id: "facebook/detr-resnet-50-panoptic",
+			description:
+				"Solid semantic segmentation model fine-tuned on ADE20k, an important benchmark comprising 150 classes.",
```
Comment (merveenoyan) [inline packages/tasks/src/tasks/image-segmentation/data.ts:48]:
```suggestion
				"Solid semantic segmentation model trained on ADE20k.",
```

**Review by merveenoyan — APPROVED** [review]
left some changes, otherwise LGTM

**Pull request merged**